fv3core/Makefile

```diff
@@ -16,7 +16,7 @@ CONTAINER_ENGINE ?=docker
 RUN_FLAGS ?=--rm
 TEST_ARGS ?=
 TEST_DATA_HOST ?=$(CWD)/test_data/$(EXPERIMENT)
-FV3UTIL_DIR=$(CWD)/external/fv3gfs-util
+FV3UTIL_DIR=$(CWD)/external/pace-util
```

Comment:
[optional] Should we change this too?

Reply:
There are a bunch of references to fv3util too when doing `import pace.util as fv3util`. Not sure what we want to do with these.